lms/envs/production.py

```diff
@@ -1048,3 +1048,9 @@ def get_env_setting(setting):
 
 ############## XBlock extra mixins ############################
 XBLOCK_MIXINS += tuple(XBLOCK_EXTRA_MIXINS)
+
+################# Settings for Chrome-specific origin trials ########
+# Token for "Disable Different Origin Subframe Dialog Suppression" Chrome Origin Trial, which must be origin-specific.
+CHROME_DISABLE_SUBFRAME_DIALOG_SUPPRESSION_TOKEN = ENV_TOKENS.get(
```

Comment:
Now we know we don't need this clause, I lead you astray. :(  But we can merge as-is if you are running out of time for this.

Reply:
I'll merge as-is, since it will mirror what's on edx-platform `master` - thanks!